src/sentry/models/useremail.py

```diff
@@ -140,3 +148,18 @@ def write_relocation_import(
         # `--merge_users=true` case is handled in the `normalize_before_relocation_import()` method
         # above).
         return (useremail.pk, ImportKind.Inserted)
+
+    @classmethod
+    def find_billing_emails_for(cls, organization_id: int) -> List[UserEmail]:
+        from sentry.models.organizationmembermapping import OrganizationMemberMapping
+
+        admin_and_owners = OrganizationMemberMapping.objects.filter(
+            role__in=["admin", "owner"], organization_id=organization_id
```

Comment:
Should this be `role__in=["billing", "owner"]`

Reply:
Good call out -- I was told "admin", "owner", so I just triple check this.